test/collections/VectorListTest.ooc

```diff
@@ -225,7 +225,13 @@ VectorListTest: class extends Fixture {
 			list free()
 		})
 		this add("VectorList sort", func {
-			//TODO Current way of sorting not supported by Rock
+			list := VectorList<Int> new()
+			list add(8); list add(16); list add(32)
+			sortedList := list copy()
+			sortedList sort(|v1, v2| v1 < v2)
```

Comment:
Well then, I guess https://github.com/fasterthanlime/rock/issues/923 is still relevant.